include/sdf/JointAxis.hh

```diff
@@ -37,6 +38,93 @@ namespace sdf
   struct PoseRelativeToGraph;
   template <typename T> class ScopedGraph;
 
+  /// \brief Helper class to hold contents of the mimic joint tag.
+  /// The parent and child joint positions are related with:
+  /// multiplier * (parentJoint - reference) + offset = childJoint.
+  class SDFORMAT_VISIBLE MimicJointContainer
```

Comment:
I've updated some of the terminology in the proposal in https://github.com/gazebosim/sdf_tutorials/pull/62 and would like to update the terminology in this pull request to match

* `MimicJointContainer` -> `MimicConstraint`
* "parent and child joint" -> "leader and follower joint axis"

Reply:
I've renamed the `MimicJoint*` classes and APIs in https://github.com/gazebosim/sdformat/pull/1166/commits/4b4097475f9473094a52525b5ca5a2bb8497f2a5

Reply:
I've updated the documentation to use follower / leader as well